app/views/logs/_form.html.erb

```diff
@@ -64,9 +64,10 @@
 
 <% unless @log.activated? %>
   <div class="row">
-    <div class="columns">
-      <span class="auto-save-indicator" style="float: right; display: none;"> Auto-Saving... </span>
+    <div class="columns" style="position: relative;">
       <%= link_to fa_icon("ban", text: "Cancel Log"), log_path(@log), method: :delete, data: { confirm: "Are you sure?" }, class: 'alert-link' %>
+      <%= button_tag "Save Draft for Later", class: 'autosave_link', data: { draft_url: save_draft_log_path(@log) }, style: "float: right; color: #fefefe; background-color: #2199e8; padding: 10px 20px; font-size: 0.9rem;", onmouseover: "this.style.backgroundColor='#1a7ec8'", onmouseout: "this.style.backgroundColor='#2199e8'" %>
```

Comment:
Please use CSS only for styling. You should very rarely, if ever, use javascript for styling. It's also preferred to extract any styling into `app/assets/stylesheets`, especially when chaining a but of style attributes together.